frontend/assets/style/_utilities.scss

```diff
@@ -62,24 +62,24 @@ $fills: (
 
 .hide-desktop {
   @include desktop {
-    display: none;
+    display: none !important;
```

Comment:
This should not happen. Could you tell me why you did this? I would like to check that this is not an error from somewhere else